scipy/stats/_discrete_distns.py

```diff
@@ -937,6 +937,21 @@ def _stats(self, a):
     def _entropy(self, a):
         return a / sinh(a) - log(tanh(a/2.0))
 
+    def _rvs(self, a):
+        # The discrete Laplace is equivalent to the two-sided geometric
+        # distribution with PMF:
+        #   f(k) = (1 - alpha)/(1 + alpha) * alpha^abs(k)
+        # Furthermore, the two-sided geometric distribution is
+        # equivalent to the difference between two iid geometric 
+        # distributions.
```

Comment:
Would be nice to add a reference, if possible.

Reply:
I can do this when I update the docstring and examples, either now or in a DOC pull request.